extensions/shared/components/upgrade-nudge/index.jsx

```diff
@@ -90,13 +102,19 @@ export default compose( [
 				redirect_to,
 			} );
 
+		const planName = get( plan, [ 'product_name' ] );
 		return {
-			trackEvent: blockName =>
+			trackViewEvent: () =>
+				void analytics.tracks.recordEvent( 'jetpack_editor_block_upgrade_nudge_impression', {
```

Comment:
I see the event being triggered in wp-admin (by adding `localStorage.setItem( 'debug', 'dops:analytics' );` to my console), but I can't seem to see it in Tracks Live view. I don't see `jetpack_editor_block_upgrade_click` either, fwiw. I am not sure why. Tracking seems to be on for my wpcom account, and I do see other events coming through such as Jetpack dashboard views. Maybe there is something wrong with the current implementation within the blocks?